api/awsauth/awsauth/welcome_email.html

```diff
@@ -19,10 +19,9 @@
       reminder, a commercial license is required for any user who engages in
       commercial activities including selling goods or services. If this
       describes you, or if you’re interested in paid support, please
-      <a href="https://calendly.com/c/CBAXPVVOF7W6SWFN?month=2020-09"
-        >sign up for a 20 minute slot</a
-      >
-      to chat with our partnerships team.
+      <a href="mailto:tommacmullen@covidactnow.org">email our Partnerships Team</a>
```

Comment:
Email address and link text don't match up. How about an email alias or mention Tom in the text?

Reply:
👍 I'm going to get some updated copy and will update soon - probably tomorrow